### ISSUE TITLE
README file

### ISSUE THREAD

**Issue by zhangzhiru** [danluhuang/bibtexDatabase#1]
Can we add some basic instructions in the README file so I can quickly reproduce your experiments?

**Comment by danluhuang** [comment]
Definitely. As soon as I finish my four finals these two days, I will work
on that. Also I will make some modifications on the code to convert the
output to a txt file instead of just printing.

Best regards,
_Danlu (Athena) Huang_

College of Engineering
Cornell University, Class of 2018
dh527@cornell.edu
